README.md

```diff
@@ -1,12 +1,135 @@
+- `npm run db:delete` deletes all existing collections in the database
+- `npm run db:reset` runs both of the above, deleting all current collections and then creating new, empty collections
+
+### CRUD Operations
+
+The test server supports the standard CRUD operations to create, read, update, and delete documents in the database. See the [MongoDB CRUD Operations documentation](https://docs.mongodb.com/manual/crud/) for more information.
+
+### Searches
+
+The test server's Measure searching capabilities support searches by name, version, both, or neither.
+
+- `GET http://localhost:3000/4_0_0/Measure/` returns all Measures currently in the database
+- `GET http://localhost:3000/4_0_0/Measure?name=NAME` returns all Measures in the database with name `NAME`
+- `GET http://localhost:3000/4_0_0/Measure?NAME&version=VERSION` returns all Measures in the database with name `NAME` and version `VERSION`
+
+The test server's resource searching capabilities support searches by identifier.
+
+- `GET http://localhost:3000/4_0_0/RESOURCETYPE/?identifier=ID` returns bundle entries of resource type `RESOURCETYPE` that have the identifier `ID`. Errors are thrown when an identifier is not included. An empty searchset is returned if the identifier cannot be found in the database.
+
+### Supported FHIR Operations
+
+#### `$evaluate-measure`
+
+This operation calculates a measure for a given patient. Currently, individual measure reports are supported. Subject-list and population measure reports are not yet supported.
+
+Required parameters include:
+
+- `periodStart`: start of the measurement period
+- `periodEnd`: end of the measurement period
+- `subject`: subject for which the measure will be calculated
+
+Currently, `measure`, `practitioner`, and `lastReceivedOn` parameters are not supported by the test server.
+
+To use, first POST a measure bundle into your database, then send a GET request to `http://localhost:3000/4_0_0/Measure/<your-measure-id>/$evaluate-measure` with the required parameters.
+
+#### `$care-gaps`
+
+This operation calculates gaps in care for a given patient against the given measure.
+
+Required parameters include:
+
+- `periodStart`: start of the measurement period
+- `periodEnd`: end of the measurement period
+- `subject`: subject for which the measure will be calculated
+- `status`: status of the care gap
+
+Currently, `topic`, `practitioner`, `organization`, and `program` are not supported by the test server.
+
+To use, first POST a measure bundle into your database, then send a GET request to `http://localhost:3000/4_0_0/Measure/$care-gaps` with the required parameters.
+
+#### `$data-requirements`
+
+This operation retrieves all the data requirements for a given measure as a FHIR library. To use, first POST a measure bundle into your database, then send a GET request to `http://localhost:3000/4_0_0/Measure/<your-measure-id>/$data-requirements`.
+
+#### `$submit-data`
```

Comment:
Suggest adding a link to the operation spec: https://www.hl7.org/fhir/measure-operation-submit-data.html